ops/docker/Dockerfile.packages

```diff
@@ -4,9 +4,9 @@
 # ### BASE: Install deps
 # We do not use Alpine because there's a regression causing it to be very slow
 # when used with typescript/hardhat: https://github.com/nomiclabs/hardhat/issues/1219
-FROM node:16.13-buster-slim as base
+FROM node:16-alpine as base
```

Comment:
Note the above comment re: Alpine regression. Is this still an issue?

Reply:
I haven't observed this issue myself.

Reply:
Will remove the comment.

Reply:
On second thought - I'll change it back. I don't see a big enough reason to change this as well, and it could introduce other issues.